README.md

```diff
@@ -64,10 +56,7 @@ It requires a Python 2.7 environment already installed.
 
 
 ## Liota.conf
-Right now there is only one item in the liota.conf, where to find a file called logging.json which holds the
-dafault initialization parameters for logging. When initialing, liota looks in the current
-working directory, '.', the user's home directory '~', a LIOTA_CONF environment variable, and
-finally the default location for every install, /etc/liota/conf for liota.conf.
+Right now there is only one item in the liota.conf, where to find a file called logging.json which holds the dafault initialization parameters for logging. When initializing, liota looks in the current working directory, '.', the user's home directory '~', a LIOTA_CONF environment variable and finally the default location for every installation: /etc/liota/conf for liota.conf.
 
 Here is the default, v0.7, liota.conf file
```

Comment:
Change this to the current version, 0.2 and also change the below config file to the latest one.